pkgs/applications/editors/leo-editor/default.nix

```diff
@@ -1,28 +1,65 @@
-{ stdenv, pythonPackages, fetchgit }:
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper, makeDesktopItem }:
 
-pythonPackages.buildPythonApplication rec {
+stdenv.mkDerivation rec {
   name = "leo-editor-${version}";
-  namePrefix = "";
-  version = "5.1";
+  version = "5.3";
 
-  src = fetchgit {
-    url = "https://github.com/leo-editor/leo-editor";
-    rev = "refs/tags/Leo-${version}-final";
-    sha256 = "0km5mvzfpfbxxhcjr4if24qhgk2c7dsvmfinz0zrbfriip848vcp";
+  src = fetchFromGitHub {
+    owner = "leo-editor";
+    repo = "leo-editor";
+    rev = version;
+    sha256 = "0whbay8ilabzpxdjaxv447y6bqbsilx161fv7wa15v3qqm2kapsp";
   };
 
-  propagatedBuildInputs = with pythonPackages; [ pyqt4 sqlite3 ];
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" "distPhase" ];
```

Comment:
Pleaser consider disabling the `buildPhase` (as in `dontBuild = true` or `buildPhase = ":"`) instead of explicitly listing phases.